lib/features/welcome/views/welcome_page.dart

```diff
@@ -1,68 +1,70 @@
-                  children: [
-                    Text("cześć"),
-                    ElevatedButton(
-                      onPressed: () {
-                        setState(() {
-                          value = "123";
-                        });
-                        Navigator.of(context).pop();
-                      },
-                      child: Text("Wyjście"),
-                    ),
-                  ],
-                ),
-              ),
-            ),
-          );
-        });
-    setState(() {
-      value = (int.parse(value) + 1).toString();
-    });
-  }
+
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: kBackground2Color,
       appBar: AppBar(
-        title: Text("WELCOME SCREEN"),
+        title: Text("KAHOOTIFY.LY"),
+        backgroundColor: kBackground1Color,
+        actions: <Widget>[
+          Padding(
+            padding: EdgeInsets.only(right: 20.0),
+            child: GestureDetector(
+              onTap: () => Navigator.of(context).push(MaterialPageRoute(builder: (context) => SettingsPage())), /*settingsDialog(context),*/
+              child: Icon(Icons.settings),
+            ),
+          )
+        ],
       ),
       body: SafeArea(
         child: Center(
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              Text(
-                "Welcome",
-                style: TextStyle(fontSize: 40, color: Colors.red),
-              ),
-              FancyButton(
-                onPressed: () => changeToTwenty(context),
-                value: value,
-              ),
-            ],
+          child: SingleChildScrollView(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Text(
+                  "SELECT GAME MODE: ",
+                  style: TextStyle(fontSize: 25, color: kButtonColor),
+                ),
+                SizedBox(height: 35),
+                Column(
+                    mainAxisAlignment: MainAxisAlignment.spaceAround,
+                    children: [
+                      WelcomeButton(
+                        /* onPressed: () => changeToTwenty(context),*/
+                        onPressed: () {
+                          print("User wybrał host-player mode");
+                        }, //TODO funkcja od wyboru host-player mode
+                        value: "HOST-PLAYER MODE",
+                      ),
+                      SizedBox(height: 50),
+                      WelcomeButton(
+                        /* onPressed: () => changeToTwenty(context),*/
+                        onPressed: () {
+                          print("User wybrał player mode");
+                        }, //TODO funkcja od wyboru player mode
+                        value: "PLAYER MODE",
+                      ),
+                      SizedBox(height: 50),
+                    ]),
```

Comment:
put ',' between ] and ) for correct code formatting

Reply:
Ready